app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +33,12 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
```

Comment:
It should be documented how to use this "API" and app shortcuts. Maybe write a wiki page and link it from README.md?

Reply:
Sure, just didn't want to go too much ahead :-D